easybuild/easyconfigs/s/ScientificPython/ScientificPython-2.9.4-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'ScientificPython'
+version = '2.9.4'
+
+homepage = 'https://sourcesup.cru.fr/projects/scientific-py/'
+description = """ScientificPython is a collection of Python modules for scientific computing.
+ It contains support for geometry, mathematical functions, statistics, physical units, IO, visualization,
+ and parallelization."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://sourcesup.cru.fr/frs/download.php/file/4570']		
+sources = [SOURCE_TAR_GZ]
+
+pyver = '2.7.11'
+pyshortver = '.'.join(pyver.split('.')[:2])
+
+versionsuffix = '-Python-%s' % pyver
+
+dependencies = [
+    ('Python', pyver),
+    # older version of numpy than the one included in Python module required
+    # cfr. https://bitbucket.org/khinsen/scientificpython/issue/13/numpy-19-has-dropped-support-for
+    ('numpy', '1.10.4', versionsuffix),
```

Comment:
@jrbosch I don't think ScientificPython is compatible with numpy 1.10.4, see the bug issue mentioned in the comment above...

I think you need to stick to the latest numpy 1.8.x, i.e. 1.8.2